### DISCUSSION TITLE
Replacing existing playlist does not work ?

### DISCUSSION THREAD

**Discussion by fillods** [rern/rAudio#843] [Fixed Issues] [answered]
When attempting to save a playlist, in the "Save current playlist as" modal window I fill in the same name as the existing playlist because I want to overwrite it. Then comes another modal window telling "Already exists" with the choice to "Replace". Up to this point, everything's fine.
Clicking "Replace" just comes with the same window, and playlist is not saved.

Here is what the clicking "Replace" triggers as HTTP request `POST mpdplaylist.php` :

```json
{
	"cmd": "save",
	"name": "Mes+webradios"
}
```
the HTTP response is ```-1```, just that, as far as what tells the WebDevTools of my FF/Linux browser.

Entering a non-existing name allows me to save the playlist.

On the board, rights and such look fine in the directory `/srv/http/data/playlists/`.

Looking at `journalctl -xeu mpd.service` showed this error message:
```syslog
rAudio mpd[10468]: exception: Failed to access /srv/http/data/mpd/playlists: No such file or directory
```
Indeed the `/etc/mpd.conf` file was wrong. I changed the `playlist_directory` from `/srv/http/data/mpd/playlists` to `/srv/http/data/playlists`.
But replacing an existing playlist still does not work.

Note: the playlist is composed of 7 _webradios_.

BTW: should be another ticket: It looks like entering a simple quote `'` in a playlist name makes the web browser barking out (invalid json), for example `L'Autre radio`.

Running:
> rAudio 1 • 20220225
> 5.11.2-1-ARCH armv7l
> TI AM335x BeagleBone Black
> AM3358 • 485MiB
> Cortex-A8 @ 1 GHz

**Comment by rern** [comment, marked as answer]
Fixed.
